### PR TITLE
movie: Starship Troopers: Traitor of Mars (2017) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
The series is over... for now. Rating: 1/5.

### CODE REVIEW

**Comment by paul-sheridan** [comment]
Hum... seems adding files to preprod after submitting a pull request somehow generates conflicts.

File: notes/stories/film/film-scifi-2010s.st.txt [hunk 1]
```diff
@@ -2555,6 +2555,59 @@ intelligent ape
...
+the military professions [One main storyline follows Colonel Johnny Rico as he and his squad of new recruits battle bug aliens on the Federation Mars colony.]
+colonization of Mars [Mars, which in this story is the main theater of war between the Federation and the bug aliens, had been terraformed and colonized by humans.]
+independence struggle [The Mars colonists were engaged in a political movement to be secede from the Federation.]
+the good of the many vs. the needs of the few [The viewer is made to ponder whether the Federation should blow Mars colony to smithereens or stand idly by and allow to be overrun by bug aliens and used by them as a stepping stone to conquering Earth.]
+
+:: Minor Themes
+terraforming [Mars had been terraformed to have a breathable atmosphere.]
+space station [The Mars Orbital Base was a large space station in orbit around Mars called.]
+speculative spaceship [The viewer is shown numerous various Federation battle cruisers over the course of the film. Captain Carmen Ibanez was shown on the bridge of the starship John A. Warden.]
+colonization of the Moon [There was a thriving spaceport on the Moon.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2010s.st.txt:2595]: Sounds like colonisation of Mars was also on the agenda..?

**Review by odinlake — APPROVED** [review]
Note comments in diff.. but feel free to merge.

File: notes/stories/film/film-scifi-2010s.st.txt [hunk 2]
```diff
@@ -2555,6 +2555,59 @@ intelligent ape
...
+action
+
+:: Major Themes
+insectoid [Federation forces were battling bug aliens, known as Arachnids.]
+extraterrestrial being [Federation forces were battling bug aliens, known as Arachnids.]
+interstellar war [The story is set in a future where humans are mired in a multiple star systems spanning war with a race of hostile bug aliens, known as the Arachnids.]
+the military professions [One main storyline follows Colonel Johnny Rico as he and his squad of new recruits battle bug aliens on the Federation Mars colony.]
+colonization of Mars [Mars, which in this story is the main theater of war between the Federation and the bug aliens, had been terraformed and colonized by humans.]
+independence struggle [The Mars colonists were engaged in a political movement to be secede from the Federation.]
+the good of the many vs. the needs of the few [The viewer is made to ponder whether the Federation should blow Mars colony to smithereens or stand idly by and allow to be overrun by bug aliens and used by them as a stepping stone to conquering Earth.]
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt:2589]: Indeed. I actually see we have **destroying one people to save another**. That fits the situation here. Just use that one?

File: notes/stories/film/film-scifi-2010s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt:2595]: Darn straight, **colonization of Mars** is up there among the major themes.

File: notes/stories/film/film-scifi-2010s.st.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt:2589]: Also I see my comment is typo ridden. Should be "The viewer is made to ponder whether the Federation should blow its Mars colony to smithereens or stand idly by and allow it to be overrun by bug aliens and used by them as a stepping stone to conquering Earth."

[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt]: Done and don.

**Pull request merged**